draft-ietf-detnet-dp-sol-mpls-02.xml

```diff
@@ -1113,37 +1004,337 @@ Examples:
+     incoming app-flows.  Note that choice to use platform label space
+     for S-Label or S-Label plus one or more F-Labels to identify app
+     flows is a local implementation choice, with one caveat.  When one
+     or more F-labels, or incoming interface, is needed together with an
+     S-Label to uniquely identify, the controller plane MUST ensure that
+     incoming DetNet MPLS packets arrive with the needed information
+     (F-label(s) and/or incoming interface); the details of such are
+     outside the scope of this document.
+   </t>
    <t>
-    The S-Label for a specific DetNet flow is unique to that DetNet flow on a specific node, but
-    is not required to be identical with the S-Label for that DetNet flow in any other node
-    within the DetNet domain. Thus the S-Label can only be used to identify the DetNet flow
-    at the intended receiving node.
+     While NOT REQUIRED, the use of platform labels for S-Labels matches
+     other pseudowire encapsulation.  This implementation choice also
+     impacts PEF and POF processing as described in the next section.
    </t>
+   <section anchor="pef-requirements" title="Packet Elimination Function
+                                             Processing">
+     <t>
+       Implementations MAY support the Packet Elimination Function (PEF)
+       for received DetNet MPLS flows.  When supported, use of the PEF
+       for a particular app flow MUST be provisioned via configuration
+       and the controller plane, see <xref target="cp_considerations"/>.
+     </t>
+     <t>
+       After an app flow is identified for a received DetNet MPLS
+       packet, as described above, an implementation MUST check if PEF
+       is configured for that app flow.  When configured the
+       implementation MUST track the sequence number contained in
+       received d-CWs and MUST ensure that duplicate (replicated)
+       instances of a particular sequence number are discarded.  The
+       specific mechanisms used for an implementation to identify which
+       received packets are duplicates and which are new is an
+       implementation choice.  Note that per <xref target="dn-sn"/>
+       the sequence number field length may be 16 or 28 bits, and the
+       field value can wrap.
+     </t>
+     <t>
+       Note that an implementation MAY wish to constrain the maximum
+       number sequence numbers that are tracked, on platform-wide or per
+       flow basis.  Some implementations MAY support the provisioning of
+       this number on either a platform-wide or per flow basis.
+     </t>
+   </section>
+   <section anchor="pof-requirements" title="Packet Ordering Function
+                                             Processing">
+     <t>
+       A function that is related to PEF is the Packet Ordering Function
+       (POF).  In addition to PEF, implementations MAY support POF. When
+       supported, use of the POF for a particular app flow MUST be

```

Comment:
POF can be also a stand alone function. This text binds it with PEF ("In addition to PEF ...").